schema/crdb/README.adoc

```diff
@@ -68,20 +85,21 @@ Process:
 SQL Validation, via Automated Tests:
 
 * The `SCHEMA_VERSION` matches the version used in `dbinit.sql`
-* The combination of all `up.sql` files results in the same schema as `dbinit.sql`
+* The combination of all `up.sql` files results in the same schema as
+  `dbinit.sql`
 * All `up.sql` files can be applied twice without error
 
-==== Handling common schema changes
+==== General notes
```

Comment:
This is kind of a vague heading but the one that was here didn't seem to describe the contents of this section.

Reply:
Maybe "Writing Schema Change Statements"? This appears to be advice for making particular statements possible, assuming that the reader already understands the high-level flow of how they're applied.